spec/features/log_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
```

Comment:
Your clear separation of feature tests in individual spec files makes it easy to read and understand.